earthpy/tests/test_plotrgb.py

```diff
@@ -0,0 +1,101 @@
+"""Tests for plot_rgb function"""
+
+import numpy as np
+import numpy.ma as ma
+import matplotlib.pyplot as plt
+import pytest
+import rasterio as rio
+from earthpy.plot import plot_rgb
+from earthpy.io import path_to_example
+
+plt.show = lambda: None
+
+
+@pytest.fixture
+def rgb_image():
+    """Fixture holding an RGB image for plotting"""
+    with rio.open(path_to_example("rmnp-rgb.tif")) as src:
+        rgb = src.read()
+    return rgb
+
+
+def test_1band(rgb_image):
+    """Test to ensure the input image has 3 bands to support rgb plotting"""
+    rgb_1band = rgb_image[1]
+
+    with pytest.raises(Exception):
```

Comment:
I see - in this case it seems like a `ValueError` might be more specific than a generic Exception (https://docs.python.org/3/library/exceptions.html#ValueError):

    Raised when an operation or function receives an argument that has the 
    right type but an inappropriate value

So you could change the `plot_rgb` code to raise a `ValueError` instead of `Exception`, but that's not totally necessary now that you've used the `match` argument. Up to you!

Reply:
Ok @mbjoseph !!! i've updated the ERROR in the function to be a ValueError and then modified the test accordingly. I'm going to see if there is anything else that i missed here... but we might be ready for another review...